compute/disks/create_disk_from_snapshot.go

```diff
@@ -29,12 +29,14 @@ import (
 func createDiskFromSnapshot(
 	w io.Writer,
 	projectID, zone, diskName, diskType, snapshotLink string,
+	diskSizeGb int64,
 ) error {
 	// projectID := "your_project_id"
-	// zone := "europe-central2-b"
+	// zone := "us-west3-b"
```

Comment:
Do developers have to use `us-west3-b`? If not, I'd make sure that this is indicated in the comments.

Reply:
Done